nextcloud_update.sh

```diff
@@ -53,21 +57,37 @@ then
     then
         if ! lvremove /dev/ubuntu-vg/NcVM-snapshot -y
         then
+            nextcloud_occ maintenance:mode --off
+            if [ -n "$DOCKER" ]
+            then
+                check_command systemctl start docker
```

Comment:
Btw, we have `start_if_stopped docker`, just remembered now....

Please look for all start occurrences and change accordingly.

Thanks!